web-api/src/persistence/postgres/cases/getCaseByDocketNumber.ts

```diff
@@ -32,13 +32,11 @@ export const getCaseByDocketNumber = async ({
     throw new NotFoundError(`Case ${docketNumber} not found`);
   }
 
-  const [caseCorrespondences, workItems, caseItemsRaw] = await Promise.all([
+  const [caseCorrespondences, docketEntries, caseItemsRaw] = await Promise.all([
     getCaseCorrespondenceByDocketNumber({
       docketNumber,
     }),
-    getWorkItemsByDocketNumber({
```

Comment:
The only reason we fetched work items here was to attach some information to docket entries. The approach in this PR is to _not_ fetch work items unless we need them, and attach the information we need when we need to.